lv_conf_template.h

```diff
@@ -581,380 +625,404 @@
     #define LV_USE_CALENDAR_CHINESE 0
 #endif  /*LV_USE_CALENDAR*/
 
+/** Include ./src/widgets/lv_canvas* logic in compile. */
```

Comment:
Why are these needed?

Reply:
Fair question.  I guess at that moment I thought it would be a good idea, but in retrospect, I think they are probably self-explanatory.  I will go ahead and remove them.

Reply:
> Regarding __lv_conf.h__ please keep it as it is. I'll take care of it separately.

Okay, I will remove that from the PR now.

Reply:
> Would you like me to remove them?

It's a trade off. If we add too many comments the config file will be noisy. With to little info, it will be hard to understand. In this particular case I vote for removing these. It should be obvious that `LV_USE_CANVAS` is used in `lv_canvas.c/h`.

Reply:
Agreed.